pkg/encoder_test.go

```diff
@@ -43,7 +43,8 @@ func TestEncoder_EncodeRaw(t *testing.T) {
 		},
 		Body: []byte{41, 42, 43, 44, 45},
 	}
-	size := e.SizeAfterEncodeRaw(group, book, rawMsg, 0)
+	sizeAfterEncode := e.SizeAfterEncodeRaw(group, book, rawMsg, 0)
+	msgSize := transport.SizeEncodedRaw(group, book, rawMsg)
```

Comment:
question: Why do we mix everything in one test? I could understand why you have added `e.SizeAfterEncodeRaw` call since it depends on the encoder state. But `transport.SizeEncodedRaw` does not depend on the encoder's state - can we introduce separate tests for newly created methods?